README.md

```diff
@@ -39,6 +39,34 @@ pnpm start
 
 Changes made to any source files in any package will make the demo rebuild and refresh. Since the projects are in the same repository and we use pnpm, we don't have to link them together.
 
+### Testing
+
+All new unit tests for components should be written using Jest and [React Testing Library](https://testing-library.com/), and not Enzyme which is being phased out.
```

Comment:
Do we need to attach the link to Jest as well? https://jestjs.io/

Reply:
I think we should to be consistent, good point Senpai :eyes: